server/controllers/food.js

```diff
@@ -101,9 +114,17 @@ export default {
    * Update a food item
    */
   async updateItem(req, res) {
+    authorizeByRole(req.user.roles, [INVENTORY, SCHEDULE])
+
     const originalCategoryId = req.params.foodId
     const updatedItem = req.body
 
+    const involvesCategoryChange = !updatedItem.categoryId || updatedItem.categoryId === originalCategoryId
+    if (involvesCategoryChange) {
+      const cantInventory = !intersection(req.user.roles, [ADMIN_ROLE, INVENTORY]).length
```

Comment:
Can't it just be `if (involvesCategoryChange) authorizeByRole(req.user.roles, [INVENTORY])`?

Reply:
Yes it can. Changed.